### PR TITLE
docs: update README

### PR DESCRIPTION
**Pull request by domire8**
## Description

- #98 

I don't think this needs a lot more than that.

## Review guidelines

Estimated Time of Review: 3 minutes

## Checklist before merging:

- [ ] Confirm that the relevant changelog(s) are up-to-date in case of any user-facing changes

### CODE REVIEW

File: .github/workflows/reusable-build-push.yml [hunk 1]
```diff
@@ -75,7 +75,7 @@ jobs:
 
           VERSION_TAG="v${{ needs.check-version.outputs.version }}"
           IMAGE_TAGS="${VERSION_TAG},${VERSION_TAG}-${{ inputs.ros_distro}}","${{ inputs.ros_distro}}"
-          GIT_TAG="${{ inputs.ros_distro}}-${VERSION_TAG}"
+          GIT_TAG="${VERSION_TAG}-${{ inputs.ros_distro}}"
```
[the remark this answers is not in the document]
Comment (LouisBrunner) [reply .github/workflows/reusable-build-push.yml:78]: Either work for me, I guess adding distro is a bit more descriptive even if not strictly semver.

File: README.md [hunk 2]
```diff
@@ -4,42 +4,31 @@
 
 ## Images
 
-### ROS workspace (noetic)
+This repository is linked so multiple Docker images where only one of them is still actively maintained and supported.
 
-The ros_ws image provides a ROS workspace (noetic). Build it by running the 
-[build script](ros_ws/build.sh) while in the [ros_ws](ros_ws) directory.
+### ROS2 workspace (ros2-ws)
```
Comment (eeberhard) [inline README.md:9]: nitpick but branding guidelines say it should be "ROS 2" not "ROS2"

File: README.md [hunk 3]
```diff
@@ -4,42 +4,31 @@
...
-### ROS2 workspace with pre-installed control libraries, modulo, and ROS2 control
-
-The ros2_modulo_control image provides a ROS2 workspace with [modulo](https://github.com/aica-technology/modulo)
-already built and comes with pre-installed [control libraries](https://github.com/aica-technology/control-libraries)
-and ROS2 control packages.
+5. ROS2 workspace with pre-installed control libraries, modulo, and ROS2 control: the ros2_modulo_control image provides
+   a ROS2 workspace with [modulo](https://github.com/aica-technology/modulo) already built and comes with pre-installed
+   [control libraries](https://github.com/aica-technology/control-libraries) and ROS2 control packages.
 
 ## Scripts
```
Comment (eeberhard) [inline README.md:33]: Do we want to stop supporting scripts at some point in this initiative? (i.e. put them under the ## deprecated section)

Comment (domire8) [reply README.md:33]: They still work though, we just don't advertise them anymore. I'd keep them there for now, it can be a good point of reference.

**Review by eeberhard — APPROVED** [review]

**Pull request merged**